testing/mr_pool.cu

```diff
@@ -1,8 +1,11 @@
 #include <unittest/unittest.h>
+
```

Comment:
Let's not have a newline between `<thrust/detail/config.h>` and the other Thrust headers.